app/code/Magento/Captcha/etc/frontend/di.xml

```diff
@@ -37,4 +37,12 @@
     <type name="Magento\Sales\Api\OrderManagementInterface">
         <plugin name="reset_payment_attempts_after_order_is_placed_plugin" type="Magento\Captcha\Plugin\ResetPaymentAttemptsAfterOrderIsPlacedPlugin"/>
     </type>
+
+    <type name="Magento\Checkout\Block\Onepage">
+        <arguments>
+            <argument name="layoutProcessors" xsi:type="array">
+                <item name="captcha_processor" xsi:type="object">Magento\Captcha\Block\CaptchaLayoutProcessor</item>
```

Comment:
I guess from the capthca module perspective, it's more "CheckoutLayoutProcessor" than "CaptchaLayoutProcessor".